indexer/packages/v4-protos/src/codegen/dydxprotocol/revshare/query.ts

```
@@ -40,6 +40,22 @@ export interface QueryMarketMapperRevShareDetailsResponse {
 export interface QueryMarketMapperRevShareDetailsResponseSDKType {
   details?: MarketMapperRevShareDetailsSDKType;
 }
+/** Queries unconditional revenue share details */
+
+export interface QueryUnconditionalRevShareConfig {}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)